src/index.ts

```diff
@@ -53,6 +54,13 @@ export const AdditionalSourceCodeStruct = object({
   url: string(),
 });
 
+export const ImagePathStruct = refine(
+  string(),
+  'Image path',
+  (value) =>
+    value.endsWith('.png') || value.endsWith('.jpg') || value.endsWith('.jpeg'),
```

Comment:
Can we validate that these are relative paths too? Maybe just checking for `./`?